main.go

```diff
@@ -115,10 +115,15 @@ func getPacketConfig(providerConfig string) (packet.Config, error) {
 	if config.LoadBalancerConfigMap == "" {
 		config.LoadBalancerConfigMap = defaultLoadBalancerConfigMap
 	}
+
 	if config.LoadBalancerConfigMap == "disabled" {
 		config.LoadBalancerConfigMap = ""
 	}
 
+	if loadBalancerConfigMap == "none" {
```

Comment:
I don't think this is necessary, as it already is covered in L115-117 above.

Reply:
This will be removed and we'll transition to a bool in the load balancer to disable all ConfigMap calls.